src/patterns/start-pages/index.md.njk

```diff
@@ -1,41 +1,73 @@
+
+Use a simple start page if users can start using the service without needing much context.
+
+<img src="simple-start-page.png" alt="A screenshot showing an example of a simple start page from the coronavirus shielding support service. The page includes a heading, text to explain who can use the service, and a start button labelled 'Register or update your details'.">
+
+If the start point is linked to a wider process that needs more explanation, create an ‘Apply’ section within a multipart guide. 
 
-Start pages include 4 main elements:
+For example, before a user applies for probate (the right to manage someone’s estate after they die) they need to understand whether probate is required, whether they’re the right person to apply, how the details of the will and the tax situation affect what they’re supposed to do, how to get the relevant information together and how the different outcomes affect what they'll need to do next.
 
-1. The service name: this should help people understand what your service does and whether they need to use it - [learn more about naming your service](https://www.gov.uk/service-manual/design/naming-your-service).
+It’s important that users have this context, so it makes sense to embed the start point within the multipart guide — rather than creating the start point as a separate start page, allowing users to miss the context and potentially do the wrong thing.
 
-2. A short introduction to list things that most users will need to know, for example, what your service will let them do or how much it’ll cost.
+<img src="multipart-guide-start-point.png" alt="A screenshot showing an example of a start point within a multipart guide. The page is shown as page 4 of 5 and its body text explains what the user will need to start using the service, followed by a start button labelled 'Apply now'.">
 
-3. The ‘Start now’ call-to-action button.
+The Carer’s Allowance team found that, for their service, [providing a start point within a multipart guide helped users get to the right information without feeling overwhelmed](https://dwpdigital.blog.gov.uk/2014/07/24/improving-to-the-carers-allowance-guide/).
 
-4. A list of other ways to access your service.
+## Creating a start point using Whitehall Publisher
 
-This is a set pattern, so you will not be able to customise it.
+For non-mainstream services, publishers in departments with access to the ‘Whitehall’ app can create a simple start page using the ‘detailed guide’ format.
 
-Read guidance on [what information should go on a start page](https://www.gov.uk/service-manual/design/govuk-content-transactions) in the Service Manual.
+Use this Markdown to create a button within a detailed guide:
 
-{{ example({group: "patterns", item: "start-pages", example: "default", html: true, nunjucks: true, open: false, titleSuffix: "second"}) }}
+`{button start}[Button text goes here](https://servicename.service.gov.uk/first-page-within-service){/button}`
```

Comment:
This should really be a [fenced code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) – single backticks are for inline code.

````suggestion
```
{button start}[Button text goes here](https://servicename.service.gov.uk/first-page-within-service){/button}
```
````

As inline code:
![Screenshot 2022-03-03 at 11 33 27](https://user-images.githubusercontent.com/121939/156556882-18a138d3-22ab-41f2-9909-92e4e1363af1.png)


With fenced code block:
![Screenshot 2022-03-03 at 11 33 36](https://user-images.githubusercontent.com/121939/156556877-6366a780-f7ef-4234-b5cb-55e60bc273f9.png)

Reply:
Thanks @36degrees — is there any way to disable the syntax highlighting? I think this was the reason Stephen gave a preference for single backticks:

From Stephen:
> Are you trying to publish example markdown? If so, backticks should work: https://www.freecodecamp.org/news/how-to-format-code-in-markdown/

Reply:
Good catch. You can specify the language after the first `` ``` `` – in this case as we want to opt out we can use `plaintext`:

````suggestion
```plaintext
{button start}[Button text goes here](https://servicename.service.gov.uk/first-page-within-service){/button}
```
````